dev/70_callback_wandb.ipynb

```diff
@@ -74,12 +74,14 @@
     "#export\n",
```

Comment:
<p><code>self.epoch</code> normally points to <code>self.learn.epoch</code> and is the integer of the current epoch. I think it's dangerous to use the same name for something slightly different. Maybe make it a private variable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_dev/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>You're right. Also at the end of one epoch <code>self.learn.epoch = 0</code> while we want <code>epoch=1</code> (because training losses will have been logged) or even more if we had a few phases before.</p><p>What do you think of starting all variables with <code>_wandb</code> to make it completely clear and safe: <code>_wandb_watch_called</code>, <code>_wandb_step</code>, <code>_wandb_epoch</code></p>

Reply:
That sounds good.

Reply:
Variables changed and [test run](https://app.wandb.ai/borisd13/fastai2/runs/kk938yrf?workspace=user-borisd13) successful.

Last issue is related to preventing logging during `lr_find`. Any suggestion?

Also an idea is to add `Interpretation.top_losses()` as a summary log at the end of each epoch or at the end of training (with potentially confusing matrix for classification) but we can keep that for later.

Reply:
Running `Learner.lr_find()` adds a callback `LRFinder` which then is added as an attribute of the `Learner`. I think simply testing `hasattr(learn, 'lr_finder')` should be enough to detect if we are in a `lr_find` and not log anything.

Reply:
Thanks it worked great!